language-support/ts/daml-react/hooks.test.ts

```diff
@@ -200,3 +232,138 @@ describe('useFetchByKey', () => {
     expect(result.current.queryResult).toEqual({contract, loading: false});
   });
 });
+
+describe('useStreamQuery', () => {
+  test('empty stream', () => {
+    // setup
+    const query = 'foo-query';
+    const [stream, emitter] = mockStream();
+    mockStreamQuery.mockReturnValueOnce(stream);
+    const hookResult = renderDamlHook(() => useStreamQuery(Foo, () => ({query}), [query]));
+    expect(mockStreamQuery).toHaveBeenCalledTimes(1);
+    expect(mockStreamQuery).toHaveBeenLastCalledWith(Foo, {query: query});
+    expect(hookResult.result.current.contracts).toEqual([]);
+    expect(hookResult.result.current.loading).toBe(false);
+
+    // no events have been emitted.
+    expect(hookResult.result.current.contracts).toEqual([]);
+    expect(hookResult.result.current.loading).toBe(false);
+
+    // empty events
+    act(() =>
+      void(emitter.emit('change', []))
+    );
```

Comment:
Apparently `void` is an operator. We can save lots of parentheses hence:
```suggestion
    act(() => void emitter.emit('change', []));
```

Reply:
changed all cases locally.